.flake8

```diff
@@ -89,6 +89,11 @@ per-file-ignores =
   # have <= members in tests (WPS202), including many local vars (WPS210),
   # additionally test docstrings don't need param lists (DAR, DCO020):
   tests/**.py: DAR, DCO020, S101, S404, S603, WPS202, WPS210, WPS436, WPS442, WPS450
+  # (B950) is ignored due to it erroring on help text which is displayed in the API.
+  # While they can be shorter, they are not a priority to shorten.
+  #(WPS226) is ignored as it is erroring on the inputs field.
+  # This current pattern is used across all plugins, we might consider extending this ignore to all plugins.
```

Comment:
It'd be also useful to add context on what the rule demands since, unlike pylint, flake8 does not name alphanumeric identifiers and it's usually hard to guess what's ignored by staring at the numbers.

Reply:
added in next commit.